ci/common/suite.sh

```diff
@@ -45,23 +45,23 @@ fail() {
 
 run_test() {
   local cmd="$1"
-  shift
+  test $# -gt 0 && shift
   local test_name="$1"
   : ${test_name:=$cmd}
-  shift
+  test $# -gt 0 && shift
   if ! eval "$cmd" ; then
     fail "${test_name}" "$@"
   fi
 }
 
 run_test_wd() {
   local timeout="$1"
-  shift
+  test $# -gt 0 && shift
   local cmd="$1"
-  shift
+  test $# -gt 0 && shift
   local test_name="$1"
```

Comment:
If `test $# -gt 0 && shift` fails won't `test_name` have the same value as `$cmd`? Why not use `$2` instead of  `shift`?

Also the file is missing shebang and error options that we usually set in bash scripts.